### DISCUSSION TITLE
RaspberryMatic Zentrale - Virtuelle Fernbedienung (drahtlos)

### DISCUSSION THREAD

**Discussion by friedpa** [OpenCCU/OpenCCU#2667] [🇩🇪 Fragen&Antworten]
Seit dem vorletzten SW update der RBM sehe ich im AskSinAnalyser ständig folgende Einträge:

![grafik](https://github.com/jens-maus/RaspberryMatic/assets/42890695/0f4e8731-192e-48b0-93dc-104d309d78d1)

Das angesprochene Device ändert sich in jedem Eintrag immer, im Beispiel oben _Heizkörperthermostat - Wohnzimmer links_ 
Gleichzeitig zeigt in der RaspberryMatic ein RPi Gateway immer um die 50% Auslastung an:

![grafik](https://github.com/jens-maus/RaspberryMatic/assets/42890695/2eaecbb4-f0db-45cb-acaa-a98972bd27eb)
 
Ich habe weder irgendein Programm geändert noch sonst etwas geändert.
Kann mir bitte jemand einen Tipp geben, wie ich das abstellen kann?
Vielen Dank im Voraus!

**Comment by jp112sdl** [comment]
Prüf mal deine Programme oder Middleware.

Irgendwas davon veranlasst deine Zentrale dazu, das HKT anzufunken.

Und da das jedes Mal per BURST geweckt werden muss, geht das arg zu Lasten des DC.

**Comment by friedpa** [comment]
Danke für Deinen Input.
Das "anfunken" ist aber nicht nur auf den HKT beschränkt:

![grafik](https://github.com/jens-maus/RaspberryMatic/assets/42890695/d139e259-ae4c-4d3a-b465-e69ca1e262e9)
![grafik](https://github.com/jens-maus/RaspberryMatic/assets/42890695/5d9cc685-0798-4b29-abab-d85034134f7a)
![grafik](https://github.com/jens-maus/RaspberryMatic/assets/42890695/98860067-76e3-4a18-b7f2-471934eaeba6)
![grafik](https://github.com/jens-maus/RaspberryMatic/assets/42890695/bf6dfe23-1287-485f-84cd-51f817f16683)

Ich habe sicher seit einem Jahr keine Programme mehr geändert, oder neu hinzugefügt. Und warum ist nur das RPi LAN Gateway betroffen und die Zentrale nicht. Das RPi LAN Gateway befindet sich abgesetzt in der Garage.
Muss ich vielleicht das Gateway neu aufsetzen, da es nicht mehr mit der neuesten RBM zusammenspielt? Das Gateway ist sicher auf einem SW Stand der 4-5 Jahre alt ist.

**Reply by jp112sdl** [reply]
Nee, am Gateway hat sich nix geändert.

Hat sich evtl. die Schnittstellenzuordnung geändert?
Schau mal in der Geräteliste bei den LAN Gatewayeinstellungen ob das alles passt.

Also nur mal das Beispiel 07:45:11 Uhr... da wird 2x eine Response von der CCU zum Fensterkontakt geschickt.
Das passiert nur, wenn der Kontakt die erste nicht erhalten hat und somit das Telegramm wiederholt.

**Comment by friedpa** [comment]
Folgende Tests habe ich gemacht:
Ich habe das RPi Gateway ausgeschaltet (Power Off) und die RBM verhält sich nun genauso, nur diesmal der gesamte Traffic (DC: 40%) am RBM Server selbst. Stecke ich das Gateway wieder an, verlagert sich das ganze wieder auf das Gateway (DC: 50%) und der DC des Servers geht auf 1-2% zurück. CS ist immer 0%. In der _LAN Gateway Konfiguration_ habe ich alle Geräte auf Standart und _Feste Zuordnung aufheben_.

**Comment by friedpa** [comment]
Bin noch immer am suchen.... was kann das verursachen?

![grafik](https://github.com/jens-maus/RaspberryMatic/assets/42890695/4764d0df-1dd2-4b55-b444-aadfa1ed7eb6)

**Reply by jp112sdl** [reply]
Ist der Türkontakt ein HM-Sec-SCo ?
Hast du bei dem Türkontakt eine Verzögerung konfiguriert? 
Da gibt es immer noch (und das wird sich auch nicht mehr ändern) den Enlos-Sende-Bug.

Wie man hier erkennen kann, wird sekündlich eine Quittung an den TFK gesendet, mit steigendem MessageCounter.

**Comment by friedpa** [comment]
ja, es ist ein HM-Sec-SCo und es ist eine Verzögerung konfiguriert. Danke für die Erklärung, habe ich nicht gewußt.
Gibt es das gleiche Problem auch bei einem HM-SCM-3-FM, Der ist nämlich der zweite Kandidat?

**Reply by jp112sdl** [reply]
Ist mir zumindest nicht bekannt, aber das muss nix heißen.

**Comment by friedpa** [comment]
Ich habe nun alles neu aufgesetzt, leider wieder dasselbe Problem mit dem virtuellen Fernbedienung Issue. Kann man diese Abfragen irgendwie abstellen, da es meinen DC völlig zuschaufelt.

![grafik](https://github.com/jens-maus/RaspberryMatic/assets/42890695/ed94e62f-f591-4759-953d-6292b78ace98)

Das große Segment ist die virtuelle Fernbedienung (drahtlos)

![grafik](https://github.com/jens-maus/RaspberryMatic/assets/42890695/3fb75cf8-6bba-4595-8e18-329ce4415024)

Wo könnte ich noch ansetzen? Ich habe kein einziges Programm das dieses Feature verwendet....

**Reply by Baxxy13** [reply]
Das was dir der Analyzer als "RaspberryMatic Zentrale - Virtuelle Fernbedienung (drahtlos)" anzeigt ist einfach der HM (Funk) Teil der Zentrale.
Mach doch mal für ne Stunde die Geräte, welche so häufig senden, stromlos.

**Comment by friedpa** [comment]
@Baxxy13 Das Problem ist, dass es einmal ein Fensterkontakt ist, dann wieder ein Heizkörperthermostat..... Z.B. heute Nacht ein Fensterkontakt (ohne Verzögerungseinstellung) mit 429(!) Telegrammen in 15 Minuten. Ich weiß leider nicht mehr, wann das begonnen hat und kann auch nicht nachvollziehen, was das ausgelöst hat, da ich seit einem Jahr nichts mehr an irgendwelchen Programmen geändert habe. Das einzige was ich gemacht habe ist der regelmäßige update auf die aktuelle Version der RM.
Wo könnte ich ansetzen um die Ursache zu finden?

**Reply by Baxxy13** [reply]
Ich kann dir nicht wirklich helfen, bin eher ein "HmIP-Kind". 
Zeichnest du die Telegramme als .csv auf? 
Dann häng die doch mal eine hier dran damit man sich ein besseres Gesamtbild machen kann.

**Comment by friedpa** [comment]
OK, ich werde das über die Osterfeiertage zusammentragen. Vielen Dank!
Anschauen durch mir die Telegramme derzeit über den AskSinAnalyzer. Wie ich die Telegramme ins Excel bekomme muss ich mir erst anschauen.

**Comment by friedpa** [comment]
Hallo @Baxxy13, anbei das Excel Sheet. Danke für Deine Hilfe!
[Protokollierung Messages RM_2024.03.xlsx](https://github.com/jens-maus/RaspberryMatic/files/14821818/Protokollierung.Messages.RM_2024.03.xlsx)

**Comment by friedpa** [comment]
@Baxxy13 Maybe any suggestions for me?

**Reply by Baxxy13** [reply]
Nicht wirklich, das liegt aber primär daran das deiner xlsx jede Menge an Daten fehlt.

Der XS-Analyzer kann csv-Dateien abspeichern die man dann auch über die UI herunterladen kann.
Sowas bräuchte ich um mir mal nen Überblick zu verschaffen.

**Comment by friedpa** [comment]
Vor zwei Tagen war auf einmal der "Spuk" vorbei. Ich habe aber nichts geändert....
![grafik](https://github.com/jens-maus/RaspberryMatic/assets/42890695/70287577-f0b1-4a45-9a9f-b3f5ba8bd680)
Vielen Dank für Deine Hilfe und Comments!
